core/modules/layout/layout.module

```diff
@@ -75,6 +75,14 @@ function layout_menu() {
     'type' => MENU_NORMAL_ITEM,
   ) + $base;
 
+  $items['admin/structure/layouts/settings'] = array(
+    'title' => 'Settings',
```

Comment:
Unless we plan to include more settings in this tab, it should be labeled something like "Manage layout templates". Better discoverability that way.